core/src/main/kotlin/org/jetbrains/research/testspark/core/test/kotlin/KotlinTestCompiler.kt

```diff
@@ -64,13 +64,20 @@ class KotlinTestCompiler(
                 "-cp",
                 classPaths,
                 path,
+                /**
+                 * Forcing kotlinc to save a classfile in the same place, as '.kt' file
+                 */
+                "-d",
+                workingDir
             ),
         )
 
         logger.info { "Error message: '$errorMsg'" }
 
-        // No need to save the .class file for kotlin, so checking the error message is enough
-        return Pair(errorMsg.isBlank(), errorMsg)
+        val classFilePath = path.replace(".kt", ".class")
```

Comment:
Are "dirty" file names like `MyKotlinClass.kt.kt` possible? 
If so, then "replace last" semantic is safer here.

Reply:
Unfortunately, there is no removeLast in Kotlin. Fixed.